libs/utils/utils.js

```diff
@@ -146,6 +146,7 @@ export function getLocale(locales, pathname = window.location.pathname) {
     return locale;
   }
   locale.prefix = locale.ietf === 'en-US' ? '' : `/${localeString}`;
+  [locale.region] = localeString.split('_');
```

Comment:
Does this cover all use-cases? Say we have these pathname examples:

1. `/products/photoshop`
2. `/ch_de/products/photoshop`
3. `/ro/products/photoshop`
4. `/`

Looking at the code, the full formula for `locale.region` seems to be `pathname.split('/')[1].split('_')[0]`. For the use-cases above, the results would be:

1. `'products'`
2. `'ch'`
3. `'ro'`
4. `''`

So to me it seems that examples 2 & 3 are working as expected, but I'm not so sure about 1 & 4. 4 might be ok for the purposes of `caas/utils.js`, but 1 still stands out. What would be the desired outcome?

Reply:
Thanks for catching that.  Fixed.